sda-commons-starter-example/src/test/java/org/sdase/commons/starter/example/PlatformRequirementsIT.java

```diff
@@ -0,0 +1,77 @@
+package org.sdase.commons.starter.example;
+
+import static io.dropwizard.testing.ConfigOverride.config;
+import static io.dropwizard.testing.ResourceHelpers.resourceFilePath;
+import static javax.ws.rs.core.Response.Status.OK;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.sdase.commons.server.opa.testing.OpaRule.onAnyRequest;
+
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.Response;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.sdase.commons.server.auth.testing.AuthClassExtension;
+import org.sdase.commons.server.opa.testing.OpaClassExtension;
+import org.sdase.commons.starter.SdaPlatformConfiguration;
+
+class PlatformRequirementsIT {
+
+  @Order(0)
+  @RegisterExtension
+  static final OpaClassExtension OPA_EXTENSION = new OpaClassExtension();
```

Comment:
NIT: Can we shorten the name to `OPA`?
```suggestion
  static final OpaClassExtension OPA = new OpaClassExtension();
```

Reply:
done!